listener/Dockerfile

```diff
@@ -1,161 +1,161 @@
-FROM base_image as build_image
-
-RUN apt-get update \
- && apt-get install -y -q \
-    pkg-config \
-    cmake \
-    make \
-    libprotobuf-dev \
-    python3-dev \
-    python3-pip \
-    swig \
-    software-properties-common
-
-
-# Copy openssl build artifacts from openssl_image
-COPY --from=openssl_image /usr/local/ssl /usr/local/ssl
-COPY --from=openssl_image /usr/local/bin /usr/local/bin
-COPY --from=openssl_image /usr/local/include /usr/local/include
-COPY --from=openssl_image /usr/local/lib /usr/local/lib
-
-RUN ldconfig \
- && ln -s /etc/ssl/certs/* /usr/local/ssl/certs/
-
-# Install setuptools packages using pip because
-# these are not available in apt repository.
-RUN pip3 install --upgrade setuptools
-
-ENV TCF_HOME=/project/avalon
-
-COPY VERSION /project/avalon/
-COPY ./bin /project/avalon/bin
-COPY ./config/tcs_config.toml /project/avalon/config/
-COPY ./common/cpp /project/avalon/common/cpp
-COPY ./common/python /project/avalon/common/python
-COPY ./common/crypto_utils /project/avalon/common/crypto_utils
-COPY ./sdk /project/avalon/sdk
-COPY ./listener /project/avalon/listener
-
-WORKDIR /project/avalon/common/cpp
-
-RUN mkdir -p build \
-  && cd build \
-  && cmake .. -DUNTRUSTED_ONLY=1 \
-  && make
-
-WORKDIR /project/avalon/common/python
-
-RUN echo "Building Avalon Common Python\n" \
- && make 
-
-WORKDIR /project/avalon/common/crypto_utils
-
-RUN echo "Building Avalon Common Crypto Python\n" \
- && make
-
-WORKDIR /project/avalon/sdk
-#install Avalon SDK  package.
-RUN echo "Building Avalon SDK\n" \
- && make 
-
-# Build Avalon Listener module next.
-
-WORKDIR /project/avalon/listener/
-
-RUN echo "Building avalon listener\n" \
-  && make
-
-# Build final image and install modules
-FROM base_image as final_image
-
-ENV TCF_HOME=/project/avalon
-
-WORKDIR /project/avalon/listener
-
-COPY --from=build_image /project/avalon/listener/listener_config.toml /project/avalon/listener/listener_config.toml
-COPY --from=build_image /project/avalon/config/tcs_config.toml /project/avalon/config/
-# Copy Python build artifacts
-COPY --from=build_image /project/avalon/common/python/dist/*.whl dist/
-COPY --from=build_image /project/avalon/common/crypto_utils/dist/*.whl dist/
-COPY --from=build_image /project/avalon/sdk/dist/*.whl dist/
-COPY --from=build_image /project/avalon/listener/dist/*.whl dist/
-
-# Installing wheel file requires python3-pip package.
-# But python3-pip package will increase size of final docker image.
-# So remove python3-pip package and dependencies after installing wheel file.
-RUN apt-get update \
- && apt-get install -y -q python3-pip \
- && pip3 install --upgrade json-rpc \
- && echo "Install Common Python, Crypto, Listener and SDK packages\n" \
- && pip3 install dist/*.whl \
- && echo "Remove unused packages from image\n" \
- && apt-get autoremove --purge -y -q python3-pip \
- && apt-get clean \
- && rm -rf /var/lib/apt/lists/*
+# Copyright 2020 Intel Corporation
```

Comment:
Why entire file is showing in the diff ? Some formatting got modified ?

Reply:
I guess, it could be due to the existing carriage return character (^M) in the listener Dockerfile.

Reply:
Yes @manojgop , updated .gitattributes to include Dockerfile for eol native formatting.